Documentation/contributing/development/index.rst

```diff
@@ -25,3 +25,4 @@ understand the workflow of getting these changes reviewed and merged upstream.
    debugging
    hubble
    documentation
+   season_of_docs
```

Comment:
So it's currently in the Development Guide: 
<img src="https://user-images.githubusercontent.com/1764210/159809846-a3ad7c67-07e6-4f17-be54-6630229b314f.png" width="200" height="400" />

Would it make more sense in the Community section above? /cc @joestringer